G.A.M.M.A/modpack_addons/A Few Random Fixes/gamedata/scripts/z_item_exo_device.script

```diff
@@ -0,0 +1,11 @@
+local itemExoDeviceOnGameStart = item_exo_device.on_game_start
+
+function item_exo_device.on_move()
+  -- noop
+end
+
+function item_exo_device.on_game_start()
+  itemExoDeviceOnGameStart()
+  UnregisterScriptCallback("actor_item_to_slot", item_exo_device.on_move)
+  UnregisterScriptCallback("actor_item_to_ruck", item_exo_device.on_move)
+end
```

Comment:
This issue originates here (which I think comes from "47- Powered Exos") because there is no `on_move()` function.

![6](https://github.com/Grokitach/Stalker_GAMMA/assets/152332990/93304b36-0baf-4b92-9427-9c1d8fba65e9)

`on_move()` is actually commented out:

![7](https://github.com/Grokitach/Stalker_GAMMA/assets/152332990/e533eb0d-94a8-4afa-b501-99e9a63915f5)

Even if it wasn't commented out, the `adjust_weight(obj)` call it makes is also commented out everywhere else, so I'm assuming it's not supposed to do anything and is just error noise in the log. My fix shuts the error up and then disables the callbacks completely.